hw8/todolist/src/components/Tasks/Tasks.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useEffect } from "react";
+import TaskComponent from "../Task/TaskComponent";
+import { Task } from "../../interfaces/Task";
+
+import {
+  TasksState,
+  completeTask,
+  deleteTask,
+  setTasks,
+} from "../../store/taskSlice";
+import { AppDispatch } from "../../store";
+import { useAppDispatch, useAppSelector } from "../../store/hooks";
+import { Routes, Route, useLocation } from "react-router-dom";
+import { tags } from "../../App";
+
+interface Props {
+  isCompleted: boolean;
+}
+
+export const addTaskToServer = async (newTask: Task) => {
```

Comment:
Extract server requests to the src/service/taskService classes to not to store it within the component